### PR TITLE
Adding the ability to see upcoming formats

### PR DESCRIPTION
**Pull request by sps-dtank**
I apologize for all the whitespace changes, I didn't know my linter was going to be so aggressive. I also had to do a rebase on the fork, so I really hope nothing was broken with that... Will fix if so.

### CODE REVIEW

File: Services/DiscordBot.cs [hunk 1]
```diff
@@ -243,7 +243,14 @@ private async Task DiscordMessageReceived(SocketMessage message)
             }
             else if (message.Content.ToLower().StartsWith(".help") || message.Content.ToLower().StartsWith("!help"))
             {
-                await message.Channel.SendMessageAsync(_settings.GetBotHelpString());
+                if (message.Content.ToLower().StartsWith(".help more") || message.Content.ToLower().StartsWith("!help more"))
+                {
+                    await message.Channel.SendMessageAsync(_settings.GetBotHelpMoreString());
+                }
+                else
```
Comment (sps-dtank) [inline Services/DiscordBot.cs:250]: this might've come back to life in the rebase

File: Services/DiscordBot.cs [hunk 2]
```diff
@@ -287,7 +294,7 @@ private async Task RecordScore(SocketMessage message)
                     var scoreChannel = _client.GetChannel(dmManifest.ScoreKeeperChannelId) as IMessageChannel;
                     if (scoreChannel != null)
                     {
-                        await scoreChannel.SendMessageAsync(message.Content.Replace(".report ", "").Replace("!report ", ""));
+                        await scoreChannel.SendMessageAsync(message.Content.Replace(".report ", "").Replace("!report ", "").Replace(".result ", "").Replace("!result ", ""));
```
Comment (sps-dtank) [inline Services/DiscordBot.cs:297]: same with this with the rebase

Comment (yort5) [reply Services/DiscordBot.cs:297]: These aren't crucial things, so I'll probably just merge them and then add them back in later if I need to.

**Comment by yort5** [comment]
> 

I'll wait until after Dice Fight tomorrow then merge and deploy.

**Pull request merged**